shared/domains/chelonia/internals.js

```diff
@@ -1516,7 +1516,23 @@ export default (sbp('sbp/selectors/register', {
         // we revert any changes to the contract state that occurred, ignoring this mutation
         console.warn(`[chelonia] Error processing ${message.description()}: ${message.serialize()}. Any side effects will be skipped!`)
         processingErrored = e?.name !== 'ChelErrorWarning'
-        this.config.hooks.processError?.(e, message)
+        const signingKeyId = message.signingKeyId()
+        const value = message.message()
+        const data = unwrapMaybeEncryptedData(value)
+        let innerSigningKeyId: string | typeof undefined
+        if (data?.data && isSignedData(data.data)) {
+          innerSigningKeyId = (data.data: any).signingKeyId
+        }
+        this.config.hooks.processError?.(e, message, {
+          signingKeyId,
+          get signingContractID () {
+            return getContractIDfromKeyId(contractID, signingKeyId, state)
+          },
+          innerSigningKeyId,
+          get innerSigningContractID () {
+            return getContractIDfromKeyId(contractID, innerSigningKeyId, state)
+          }
+        })
```

Comment:
Maybe put this object `msgMeta` into a function that returns the object instead, instead of inlining the object, and call the function here.

Reply:
Done making it a function